ChannelCard의 UI 완성

always-awake opened the pull request:
# ChannelCard의 UI 완성

## 해당 이슈 📎

- [Epic#31] 채널 코드를 입력할 수 있다.(#31)

## 변경 사항 🛠

- ChannelCard 컴포넌트의 UI 구현을 완료하였다.

## 테스트 ✨
> 없음

## 리뷰어 참고 사항 🙋‍♀️

> 없음

always-awake committed:
[ FEATURE ] 채널 목록에서 채널 상태에 따라 다른 스타일을 갖는 ChannelStatusBadge 컴포넌트 구현 (#50)

Why
- 사용자가 채널 목록에서 입장하고자 하는 채널의 상태를 쉽게 파악할 수 있어야 한다.

What
- channelStatus props를 받아 해당 값에 따라 다른 스타일을 렌더링하는 ChannelStatusBadge 컴포넌트를 구현하였다.

always-awake committed:
[ FEATURE ] ChannelCard에 채널 상태와 마스터 이름이 함께 렌더링 할 수있도록 수정 (#50)

Why
- ChannelCard에 꼭 필요한 채널 상태와 마스터 이름을 함께 보여주어야 한다.

What
- useGetChannelsByCode 커스텀 훅에서 master에 대한 정보를 추가로 함께 fetching 해올 수 있도록 수정하였다.
- 추가로 fetching 해온 데이터를 ChannelCard 컴포넌트가 렌더링 할 수 있도록 수정하였다.

devjaewon commented on web/src/components/main/ChannelListModal/ChannelCard/style.jsx:
```diff
@@ -1,14 +1,38 @@
 
 export default {
   ChannelCard: styled.div`
   width: 100%;
```
indent 를 맞추는게 좋지 않을까요?
css indent 가 이 컴포넌트만 다른 형태로 되어있네요!

always-awake replied:
앗 실수입니다ㅠㅠ 수정하겠습니다!

devjaewon commented on web/src/components/main/ChannelListModal/style.jsx:
```diff
@@ -9,12 +9,12 @@ export default {
     height: 100%;
-background-color: ${colorCommon('white')}BF;
-    padding-top: 64px;
+    background-color: ${colorCommon('white')}BF;
```
뒤에 있는 BF 는 어떤 뜻일까요?

always-awake replied:
투명도입니다!

devjaewon reviewed (approved):
전체적으로 팀 컨벤션을 잘 따르는 좋은 코드 같네요 👍
수고하셨습니다.

always-awake committed:
[ REFACTOR ] 변수명 오타 수정 (#50)

Why
- 원래 의도한 변수명인 refindedChannels 변수명으로 수정해야 한다.

What
- refindeChannels 변수명을 refindedChannels로 수정하였다.
